pypeit/spectrographs/lbt_luci.py

```diff
@@ -0,0 +1,492 @@
+
+        # Wavelengths
+        # 1D wavelength solution
+        par['calibrations']['wavelengths'][
+            'rms_threshold'] = 0.20  # 0.20  # Might be grating dependent..
+        par['calibrations']['wavelengths']['sigdetect'] = 5.0
+        par['calibrations']['wavelengths']['fwhm'] = 5.0
+        par['calibrations']['wavelengths']['n_final'] = 4
+        par['calibrations']['wavelengths']['lamps'] = ['OH_NIRES']
+        par['calibrations']['wavelengths']['nonlinear_counts'] = \
+            self.detector[0]['nonlinear'] * self.detector[0]['saturation']
+        par['calibrations']['wavelengths']['method'] = 'holy-grail'
+
+
+        par['calibrations']['slitedges']['edge_thresh'] = 300
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+        par['calibrations']['slitedges']['fit_order'] = 8
+
+        # Flats
+        par['calibrations']['flatfield']['illumflatten'] = True
+        # par['calibration']['flatfield']['tweak_slits'] = False
+
+        # Extraction
+        # Model full slit currently turned on
+        par['scienceimage']['extraction']['model_full_slit'] = True
+        # Tailored profile nsigma parameter for the standard
+        par['scienceimage']['extraction']['std_prof_nsigma'] = 100.
+        # Do not perform global sky subtraction for standard stars
+        par['scienceimage']['skysub']['global_sky_std'] = False
+        par['scienceimage']['skysub']['bspline_spacing'] = 0.8
+        par['scienceimage']['extraction']['sn_gauss'] = 4.0
+
+        # Flexure
+        par['flexure']['method'] = 'skip'
+
+        par['scienceframe']['process']['sigclip'] = 20.0
+        par['scienceframe']['process']['satpix'] = 'nothing'
+        # par['scienceframe']['process']['satpix'] = 'reject'
+
+        par['scienceframe']['process']['overscan'] = 'none'
+        # par['standardframe']['process']['overscan'] = 'none'
+
+        return par
+
+
+    def config_specific_par(self, scifile, inp_par=None):
+        """
+        Modify the PypeIt parameters to hard-wired values used for
+        specific instrument configurations.
+
+        .. todo::
+            Document the changes made!
+
+        Args:
+            scifile (str):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`pypeit.par.parset.ParSet`: The PypeIt paramter set
+            adjusted for configuration specific parameter values.
+        """
+        # Start with instrument wide
+        par = super(LBTLUCI2Spectrograph, self).config_specific_par(scifile,
+                                                             inp_par=inp_par)
+
+        # Try to set the detector parameters dependent on the scifile here
```

Comment:
Remove.  And it looks like you can remove this method altogether.

Reply:
Method removed.